src/main/java/com/keboola/tableexporter/MetaFetcher.java

```diff
@@ -79,7 +79,9 @@ public TreeMap fetchTableListing() throws UserException {
     public LinkedHashMap getTableData(ResultSet rs) throws SQLException {
         LinkedHashMap tableData = new LinkedHashMap();
         tableData.put("name", rs.getString("TABLE_NAME"));
-        tableData.put("tablespaceName", rs.getString("TABLESPACE_NAME"));
```

Comment:
woudn't it be better to cast it to empty string?

Reply:
why?  null means missing, right?

Reply:
right, but then you can't pick a table with no tablespacename

or we would have to need to search a table by non-presence of metadata?

Reply:
yes, I think a simple `hasKey(key)` can work with that.